WordPress/WordPressShare/BlogPickerViewController.swift

```diff
@@ -0,0 +1,135 @@
+import Foundation
+import UIKit
+import WordPressShared
+import WordPressComKit
+
+
+class BlogPickerViewController : UITableViewController
```

Comment:
Let's rename to `SitePickerViewController`, and add a comment on top of it, outlining that it doesn't work with Core Data. Otherwise the class name would look like this... `SitePickerDetachedFromCoreDataViewController`, and... it would make me really, really sad.

Thank you both!